README.md

```diff
@@ -2,10 +2,18 @@
 
 Net::Fastly - client library for interacting with the Fastly web acceleration service
 
+# A Note About Authentication
+
+Authenticating with a username/password is deprecated and will no longer be available starting September 2020.
+
+Authenticating with an API Token is shown in the example synopsis below. For more information on API Tokens, please see [Fastly's API Token documentation](https://developer.fastly.com/reference/api/auth/). For more information about authenticating to our API, please see our [Authentication section](https://developer.fastly.com/reference/api/#authentication).
+
```

Comment:
It looks like the README should be generated from the POD (inline documentation) in lib/Net/Fastly.pm.  We should make the changes there and run `pod2markdown lib/Net/Fastly.pm > README.md`.

For details on POD:

https://perldoc.perl.org/perlpod.html

but tl;dr is that this should look something like:

```
=head2 A Note About Authentication 

blah blah blah  L<Fastly's API Token documentation|https://developer.fastly.com/reference/api/auth/>
```

This should probably go under `DESCRIPTION`, before `METHODS`

The POD inside the .pm file is also what will be displayed on metacpan.org here:
https://metacpan.org/pod/Net::Fastly

Which will be seen by more people than the README.

Reply:
👍